src/CardGroup.js

```diff
@@ -0,0 +1,3 @@
+import createWithBsPrefix from './utils/createWithBsPrefix';
+
+export default createWithBsPrefix('card-group');
```

Comment:
@jquense do you reckon this is okay to do?

Reply:
Sure, though we will need to figure out how to generate docs for these cases but that's something we need to do anyway.